packages/taquito-contracts-library/test/taquito-contracts-library.spec.ts

```diff
@@ -90,19 +101,36 @@ describe('ContractsLibrary tests', () => {
           script,
           entrypoints,
         },
+    
       });
     } catch (e: any) {
+      expect
+      expect(validateAddress(contractAddress)).toEqual(0)
       expect(e).toBeInstanceOf(InvalidAddressError);
       expect(e.message).toEqual(`Address is invalid: ${contractAddress}`);
       expect(e).toBeInstanceOf(Error);
     }
   });
 
+  it('do not throw an InvalidAddress error if the contract address is valid', () => {
```

Comment:
Is it intended that there is no expect statement in the test?

Reply:
added these expectations:

Reply:
const contractData = contractLib.getContract(contractAddress);
    expect(contractData.entrypoints).toEqual(entrypoints);
    expect(contractData.script).toEqual(script);